cypress/fixtures/seed.json

```diff
@@ -37,7 +37,7 @@
                   "labels":[
                       {
                           "ver":1,
-                          "src":"did:plc:4wke3qtisoohr7tf6vfhe5r2",
```

Comment:
these had to be changed because we never used the src value for any functionality being tested, now that we're hitting that path in tests, we need to make sure the src matches with the test dataset.